corehq/apps/translations/integrations/transifex/exceptions.py

```diff
@@ -1,6 +1,4 @@
-
-
-class ResourceMissing(Exception):
+class TransifexApiException(Exception):
```

Comment:
The thing here is we can really only guess at why the request failed, since most of the time the Transifex library is just giving us a `JsonApiException`. Rather than make assumptions, I preferred to wrap the errors in something that makes it clear the issue occurred in accessing the Transifex API code.